packages/client/src/client/notifications.web.ts

```diff
@@ -99,11 +114,12 @@ export async function registerFxNotifications() {
 }
 
 let quotesReceivedSubscription: Subscription | null = null
```

Comment:
we are missing a `unregisterFxNotifications` here
.. we should use a type somehow to keep our variant modules honest ... 🤔